node/src/command.rs

```diff
@@ -50,8 +50,6 @@ impl SubstrateCli for Cli {
         Ok(match id {
             "dev" => Box::new(chain_spec::development_config()?),
             "" | "inteli-stage" => Box::new(chain_spec::inteli_stage_config()?),
```

Comment:
maybe we should call this `inteli-staging`

Reply:
it's called `inteli-stage` in the infra hence the name